ArduCopter/mode.h

```diff
@@ -3,6 +3,7 @@
 #include "Copter.h"
 #include <AP_Math/chirp.h>
 #include <AP_ExternalControl/AP_ExternalControl_config.h> // TODO why is this needed if Copter.h includes this
+#include <AP_Math/control.h>
```

Comment:
I think this could just be included in the cpp.

Reply:
Done. Thanks